CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 0.49.0 (2024-11-14)
+
+
+### Features
+* **JA**: Add manifest upload and download to complete the JA standalone API+CLI featureset. ([`b5b203b`](https://github.com/aws-deadline/deadline-cloud/commit/b5b203bf5b370cb5e51b3e60f6b052e87e11d6d7))
+* refactor manifest aggregation, add helper to persist manifests and check disk capacity (#483) ([`f57f637`](https://github.com/aws-deadline/deadline-cloud/commit/f57f637d78c2b5643a39b1c3b6190cbae745c079))
+* **JA**: Add force-rehash option to snapshot command. Move Integ Tsts for Diff and Snapshot to Integ ([`276edae`](https://github.com/aws-deadline/deadline-cloud/commit/276edaea9cc11f8a0800a9f336f9e0d2057ff53b))
+* **JA**: Implementation for Job Attachments diff command (#465) ([`97e8bc4`](https://github.com/aws-deadline/deadline-cloud/commit/97e8bc45a21b3e88bc37ecefb42b92055ecd7ecf))
+
+### Bug Fixes
+* De-duplicate error messages on expired credentials ([`cfc7759`](https://github.com/aws-deadline/deadline-cloud/commit/cfc77592edc472442950d0539ee1093cee534a59))
+* Refresh queue parameter defaults when loading a new job bundle ([`5abcef1`](https://github.com/aws-deadline/deadline-cloud/commit/5abcef1b2b88bcf189786ed7fa6ca826ee536ed6))
+* The hash_cache fails with filenames including surrogates (#492) ([`39c285d`](https://github.com/aws-deadline/deadline-cloud/commit/39c285d2e8bc3638a9f88f14ccd8c234b49cb2c7))
+* Fix skipped test. Should not hard code root prefix of Queue CAS as an assumption. (#491) ([`d9e70d2`](https://github.com/aws-deadline/deadline-cloud/commit/d9e70d252d7b47cf6934e2df4eff5201ff037e7c))
+* Integration test fix and skip a test. (#490) ([`eaef7e0`](https://github.com/aws-deadline/deadline-cloud/commit/eaef7e01c67e514727a616272954d0d9ccfe3009))
```

Comment:
These two test items shouldn't show up in the change log.

Reply:
I will remove lines 14 & 15